README.md

```diff
@@ -14,63 +14,53 @@ Use these commmands to download, import data and create tiles jobs for Luxembour
 **If sudo required, use "-E"!!**
 
 ```bash
-docker-compose -f docker-compose.yml -f local-compose.yml down -v
+./exec.py shutdown
 ```
 
 Download, import and start the tiles generation:
 
 ```bash
-docker-compose -f docker-compose.yml -f local-compose.yml up --build -d
-docker-compose -f docker-compose.yml -f local-compose.yml run --rm -e INVOKE_OSM_URL=https://download.geofabrik.de/europe/luxembourg-latest.osm.pbf -e INVOKE_TILES_X=66 -e INVOKE_TILES_Y=43 -e INVOKE_TILES_Z=7 load_db
+./exec.py load-db # it runs the build command as a dependency on 'load-db'
```

Comment:
We should explain somewhere that this command will import osm data and generate tiles on Luxembourg only, by default.

Reply:
Done in the helper.